src/scripts/main.js

```diff
@@ -357,4 +357,28 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const dashboard = document.querySelector('.dashboard');
```

Comment:
Ensure that the 'dashboard' element exists in the DOM before attempting to append rows to it. If the element is not found, 'dashboard' will be null, and calling 'appendChild' on it will result in an error. Consider adding a check to handle this case gracefully.